GW2EIParserCLI/Program.cs

```diff
@@ -17,6 +17,17 @@ private static int Main(string[] args)
             Properties.Settings.Default.Outdated = false;
         }
 
+        // Clean up temp folder on startup
+        try
+        {
+            // TODO restore this comment before merging
+            // Directory.Delete(Path.Combine(Path.GetTempPath(), Updater.EI_TempFolder), true);
+        }
+        catch
+        {
+
+        }
+
```

Comment:
The temp folder does not necessarily need cleanup, but I also don't see any reason as to why you couldn't do this in the updater after you have started the updated version.